### DISCUSSION TITLE
Один провайдер, два разных сервера на одном все ок, на другом не ок

### DISCUSSION THREAD

**Discussion by VitArts** [xtrime-ru/antizapret-vpn-docker#107] [General]
Подскажите куда копать, есть два идентичных серверах с Wireguard, на одном все работает как часы, на другом не открывает не один сайт.

Вот на этом сервере не работает:
![image](https://github.com/user-attachments/assets/0da265b6-0265-4040-a116-c89f7235927a)
![image](https://github.com/user-attachments/assets/42d399dd-d8f5-41a8-80c1-7baff10d7977)
![image](https://github.com/user-attachments/assets/f7a9e53b-b3e5-4787-baec-81ce60a5fe55)

**Comment by xtrime-ru** [comment]
На указанных скриншотах все корректно. Адрес youtube подменяется, а youtube нет. Curl так же работает.

**Comment by xtrime-ru** [comment]
Что на клиенте выдают команды ?

```
nslookup youtube.com
nslookup youtube.com 10.224.0.1
```

**Comment by VitArts** [comment]
На любой хост 
;; connection timed out; no servers could be reached

**Reply by xtrime-ru** [reply]
Значит соединение с сервером не установлено. Надо разбираться почему. Мб на сервере закрыты порты через firewall хостера. Wireguard к сожалению в этом плане неудобен, так как не проверяет свое соединение.

**Comment by VitArts** [comment]
Настроил на Amnezia, сразу все полетело.
